alot/commands/thread.py

```diff
@@ -944,13 +944,10 @@ def apply(self, ui):
                     filename = self.attachment.get_filename()
                     prefix, suffix = fn_hook(filename, prefix, suffix)
 
-                tmpfile = tempfile.NamedTemporaryFile(delete=False,
-                                                      prefix=prefix,
-                                                      suffix=suffix)
-
-                tempfile_name = tmpfile.name
-                self.attachment.write(tmpfile)
-                tmpfile.close()
+                with tempfile.NamedTemporaryFile(delete=False, prefix=prefix,
+                                                 suffix=suffix) as tmpfile:
```

Comment:
You're wrapped this and the one after in the opposite manner, from a consistancy point of view I'd prefer to see one or the other.